profiles/coreos/base/package.use

```diff
@@ -39,7 +39,7 @@ net-analyzer/nmap ncat -lua
 app-admin/sudo -sendmail
 
 # avoid pulling in gnutls, disable gentoo-only bits, enable journal upload
-sys-apps/systemd -ssl curl vanilla lzma
+sys-apps/systemd -ssl curl vanilla -lz4 lzma gcrypt
```

Comment:
We can do both, but we'll have to pull in app-arch/lz4. I don't feel strongly either way. What do you think?